internal/v2/tls_config_store/tls_config_store_test.go

```diff
@@ -141,6 +142,17 @@ func TestTLSConfigStoreClient(t *testing.T) {
 	}{
 		{
 			description:        "static",
+			tokenManager:       nil,
```

Comment:
Can we add the analogous test case for the GetTLSConfigurationForServer API?

Reply:
Will do!